src/main/java/com/wanted/preonboarding/common/model/ReservableEntity.java

```diff
@@ -0,0 +1,32 @@
+package com.wanted.preonboarding.common.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+
+@MappedSuperclass
+@Getter
+public class ReservableEntity extends DefaultEntity {
+
+    private static final String ENABLE = "enable";
+    private static final String DISABLE = "enable";
```

Comment:
```suggestion
    private static final String DISABLE = "disable";
```
무언가 작성하시다가 오타가 있으셨던거 같아요 :)